### PR TITLE
Feat: add left sidebar

### LINKED ISSUE #6: Sidebar container
## User's posts appear on their sidebar

The title of a post, in a sidebar container.

### PR DESCRIPTION
**Pull request by dewinterjack**
**update** now looking to use sidebar for navigation (@michaelfitzhavey): fixes #6 

**end update**

* Add post now in sidebar
* User's posts appear as a list
* Click links to the post

**Before:**

before

**After:**

progress

### CODE REVIEW

File: src/components/AppSidebar.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+			class="align-middle"
+			v-if="isAddButtonVisible"
+			variant="primary"
+			v-text="'Add Post'"
+			to="posts/add"/>
+		
+
+		<!-- #Todo: look at implementing sidebar items similar to: https://vuetifyjs.com/en/examples/layouts/googleKeep -->
+
+		<!-- #Todo: swap this paragraph for a <b-link> (see: https://bootstrap-vue.js.org/docs/components/link) -->
```
Comment (michaelfitzhavey) [inline src/components/AppSidebar.vue]: remove #Todo if completed

File: src/vuex/modules/postStore.js [hunk 2]
```diff
@@ -88,7 +88,6 @@ export default {
 			const postData = snap.val();
 			const post = await dispatch('buildPostForUI', { post: postData, id });
 			commit('SET_POST', post);
-			return post;
```
Comment (michaelfitzhavey) [inline src/vuex/modules/postStore.js]: can we add this line back in?

Comment (dewinterjack) [reply src/vuex/modules/postStore.js]: hmm, I didn't think I touched that

**Comment by dewinterjack** [comment]
@michaelfitzhavey  see changes 🥇

**Comment by michaelfitzhavey** [comment]
@dewinterjack looking good - what about this one? divider below button, maybe add an icon?

**Pull request merged**